### PR TITLE
thrift090 0.9.0 (fixes formulae)

### PR DESCRIPTION
**Pull request by rafaelverger**
Fixing patches URLs

### CODE REVIEW

**Comment by signalpillar** [comment]
Can this error be a problem of my environment 

```
brew install https://raw.githubusercontent.com/rafaelverger/homebrew-versions/master/thrift090.rb --with-java --with-python
######################################################################## 100.0%
==> Downloading https://archive.apache.org/dist/thrift/0.9.0/thrift-0.9.0.tar.gz
Already downloaded: /Library/Caches/Homebrew/thrift090-0.9.0.tar.gz
==> Downloading https://gist.githubusercontent.com/rafaelverger/2602feff4050df0693c5/raw/35344407630182efb679de6c29804f0334c2ad26/patch-tsocket.patch
Already downloaded: /Library/Caches/Homebrew/thrift090--patch-a1dc9e54ffacf04c6ba6d1e37b734684ff09d149a88ca7425a4237267f674829.patch
==> Downloading https://gist.githubusercontent.com/rafaelverger/2602feff4050df0693c5/raw/35344407630182efb679de6c29804f0334c2ad26/patch-cxx11-compat.patch
Already downloaded: /Library/Caches/Homebrew/thrift090--patch-dedc5fbf5fb0e1c8bba20bb4ed1a5d4bc617237c02c47a7d1eb36e8bcb41bb20.patch
==> Downloading https://gist.githubusercontent.com/rafaelverger/2602feff4050df0693c5/raw/35344407630182efb679de6c29804f0334c2ad26/patch-use-boost-cpp-client-server.patch
Already downloaded: /Library/Caches/Homebrew/thrift090--patch-2ea5a69c5358a56ef945d4fb127c11a7797afc751743b20f58dfff0955a68117.patch
==> Downloading https://gist.githubusercontent.com/rafaelverger/2602feff4050df0693c5/raw/35344407630182efb679de6c29804f0334c2ad26/patch-remove-tr1-dependency.patch
Already downloaded: /Library/Caches/Homebrew/thrift090--patch-d03c712b794a7fee0335877204bf5cb8f1c0b345c51443a9ea6e4020870828ca.patch
==> Patching
patching file lib/cpp/src/thrift/transport/TSocket.h
patch unexpectedly ends in middle of line
Hunk #1 succeeded at 21 with fuzz 1.
patching file compiler/cpp/src/thrifty.yy
patching file compiler/cpp/src/generate/t_rb_generator.cc
Hunk #1 FAILED at 352.
1 out of 1 hunk FAILED -- saving rejects to file compiler/cpp/src/generate/t_rb_generator.cc.rej
patching file lib/cpp/src/thrift/server/TNonblockingServer.cpp
patch unexpectedly ends in middle of line
Hunk #1 succeeded at 478 with fuzz 1.
Error: Failure while executing: /usr/bin/patch -g 0 -f -p1 -i /private/tmp/thrift090--patch20150912-41137-1bheaux/patch-cxx11-compat.patch
```

**Comment by dunn** [comment]
No, looks like the bot is having the same problem:
```
==> Patching
/usr/bin/patch -g 0 -f -p1 -i /private/tmp/thrift090--patch20150911-13589-1e5gaec/patch-tsocket.patch
patching file lib/cpp/src/thrift/transport/TSocket.h
patch unexpectedly ends in middle of line
Hunk #1 succeeded at 21 with fuzz 1.
/usr/bin/patch -g 0 -f -p1 -i /private/tmp/thrift090--patch20150911-13589-166c2bp/patch-cxx11-compat.patch
patching file compiler/cpp/src/thrifty.yy
patching file compiler/cpp/src/generate/t_rb_generator.cc
Hunk #1 FAILED at 352.
1 out of 1 hunk FAILED -- saving rejects to file compiler/cpp/src/generate/t_rb_generator.cc.rej
patching file lib/cpp/src/thrift/server/TNonblockingServer.cpp
patch unexpectedly ends in middle of line
Hunk #1 succeeded at 478 with fuzz 1.
Error: Failure while executing: /usr/bin/patch -g 0 -f -p1 -i /private/tmp/thrift090--patch20150911-13589-166c2bp/patch-cxx11-compat.patch
```
@rafaelverger, did this work for you locally?

**Comment by rafaelverger** [comment]
@dunn @signalpillar it was an error due to missing whitespaces in two patches but I've fixed now..

Oh... I've force changed my previous commit, I'm sorry for this; Can we still work in this PR or should I close this one and open again? The jenkins' job is stuck apparently: http://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/875

**Comment by DomT4** [comment]
> The jenkins' job is stuck apparently: http://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/875

It's not. The CI is just a little busy at the moment. The build is being allocated machines as they become available.

**Comment by rafaelverger** [comment]
Hmmm job finished with a failure in mountain_lion. Apparently a git error, may we retry the job?

**Comment by DomT4** [comment]
Heh. Jenkins is lightly trolling us there. Have re-queued it.

**Comment by rafaelverger** [comment]
Now it's ok :) @DomT4

**Comment by signalpillar** [comment]
@rafaelverger first of all, thank you for fixing this formula. 

Have you considered to keep required patches for the thrift as part of this repo not as gist, so you don't have to keep track of them in your account ?

**Comment by rafaelverger** [comment]
@signalpillar this would be great :) since I've didn't see anyone doing it I thought that wasn't allowed.

how can we do this?

**Comment by signalpillar** [comment]
@rafaelverger maybe we can create a folder in this repo like `thrift090-patches` put patch files there and refer (raw) them in the `.rb` file. This should work, IMO.

**Comment by DomT4** [comment]
Please don't dump patches in this repo. We keep the tap repos for formulae. The current format is fine. You're welcome to submit them to https://github.com/Homebrew/patches if you really don't want to look after them on your account.

**Comment by signalpillar** [comment]
@DomT4 this is a really nice option to have them in `patches` repo. Thank you!
@rafaelverger What do you think ?

**Comment by rafaelverger** [comment]
@signalpillar @DomT4 I'm moving the patches to homebrew/patches repo right now :) after this is done I will update this PR :)

**Comment by rafaelverger** [comment]
Waiting for merge in homebrew/patches https://github.com/Homebrew/patches/pull/1 :)

**Comment by rafaelverger** [comment]
@DomT4 @signalpillar done :)

File: thrift090.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ class Thrift090 < Formula
 
   bottle do
     cellar :any
+    revision 1
```
Comment (dunn) [inline thrift090.rb]: Woops, let's remove this. Bottle revisions are handled by the bot.

Comment (rafaelverger) [inline thrift090.rb]: Ok @dunn, done :)

**Comment by signalpillar** [comment]
Thanks @rafaelverger, lets when when they will be merged

**Comment by rafaelverger** [comment]
Those errors are about mountain_lion nodes that couldn't be started :S

**Comment by rafaelverger** [comment]
Anyone able to help with those build errors on mavericks?
```
==> brew install --only-dependencies --verbose --build-bottle Homebrew/versions/thrift090Build timed out (after 120 minutes). Marking the build as aborted.```

**Comment by DomT4** [comment]
There's a CI backlog we're trying to clear. Just bear with us on this.

**Comment by rafaelverger** [comment]
@DomT4 looks like u guys solved the backlog, nice!

**Comment by rafaelverger** [comment]
Can anyone look look this PR? @signalpillar @DomT4

**Comment by DomT4** [comment]
I'll merge it later tonight. Just bear with me till then.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/d0db458d70c3af2e60728408c809553146ac8add. Thank you for your contribution to Homebrew @rafaelverger!

**Comment by signalpillar** [comment]
@rafaelverger thank you very much!

**Comment by rafaelverger** [comment]
no problem @signalpillar ... thank you guys too :)

**Pull request closed without merging**